src/cli/flowr.ts

```diff
@@ -88,47 +88,58 @@ if(options['no-ansi']) {
 	setFormatter(voidFormatter);
 }
 
-let usedConfig = false;
-if(options['config-json']) {
-	const config = parseConfig(options['config-json']);
-	if(config) {
-		log.info(`Using passed config ${JSON.stringify(config)}`);
-		setConfig(config);
-		usedConfig = true;
+function createConfig() : FlowrConfigOptions {
```

Comment:
In the future we probably want to have one general create, but for now this is fine :)